.github/workflows/staging-build-deb.yml

```diff
@@ -583,6 +583,83 @@ jobs:
           working-directory: kibana/plugins/workbench
           command: npx cypress run
 
+  Test-Kibana-Notebooks-NoSec:
+    needs: [sign-deb-artifacts]
+    runs-on: [ubuntu-18.04]
+    name: Test-Kibana-Notebooks-NoSec
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [14]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up AWS Cred
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Retrieve plugin tags 
+        run: |
+          echo "p_tag_kib_nb=$(release-tools/scripts/plugin_tag.sh opendistro-for-elasticsearch/kibana-notebooks)" >> $GITHUB_ENV
```

Comment:
As this is new implementation, please pick the tag version from Manifest.

Reply:
I have not tested with new implementation. May be we do a global replacement and test all at once. That will also include separating the IT altogether in new workflow

Reply:
Sure.